frontend/app/student/my-cohorts/page.tsx

```
@@ -0,0 +1,708 @@
+    case "graded":
+      return <Badge className="bg-green-100 text-green-800 text-xs">Completed</Badge>
+    case "available":
+      return <Badge className="bg-red-100 text-red-800 text-xs">Available</Badge>
+    case "in_progress":
+      return <Badge className="bg-blue-100 text-blue-800 text-xs">In Progress</Badge>
+    case "submitted":
+      return <Badge className="bg-yellow-100 text-yellow-800 text-xs">Submitted</Badge>
+    default:
+      return <Badge className="bg-gray-100 text-gray-800 text-xs">{status}</Badge>
+  }
+}
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)